.github/workflows/gurobi-auto-update.yml

```diff
@@ -0,0 +1,73 @@
+name: Gurobi Auto Updater
+
+on:
+  schedule:
+    - cron: '15 1 * * 3' # Every Wednesday at 1:15 AM UTC
+  workflow_dispatch:
+
+jobs:
+  update-gurobi-version:
+    if: github.ref == 'refs/heads/main'
```

Comment:
```suggestion
```
I just checked again and this line is not necessary. According to https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule a `schedule` job run on "Last commit on default branch".